### PR TITLE
Adding Drift V2 endpoints part 1

### PR DESCRIPTION
**Pull request by tinystarinagalaxy**

### CODE REVIEW

File: bxsolana/provider/ws.ts [hunk 1]
```diff
@@ -102,6 +102,20 @@ import {
 import { BaseProvider } from "./base"
 import { RpcWsConnection } from "../ws/rpcclient"
 import { RpcReturnType } from "../proto/runtime/rpc"
+import { Type as GetDriftMarketsRequest } from "../proto/messages/api/GetDriftMarketsRequest"
```
Comment (aspin) [inline bxsolana/provider/ws.ts]: these imports are weird, can you fix? you should be able to `import { GetDriftMarketsRequest } from "../proto/messages/api"`

**Review by aspin — CHANGES_REQUESTED** [review]
imports need fixing

**Review by aspin — CHANGES_REQUESTED** [review]
imports need fixing

File: bxsolana/provider/ws.ts (hunk 1 above)
Comment (tinystarinagalaxy) [reply bxsolana/provider/ws.ts]: this is fixed everywhere now

**Review by aspin — APPROVED** [review]

**Pull request merged**